lib/commands/ls.js

```diff
@@ -23,5 +23,6 @@ class LsCommand extends Command {
 
 LsCommand.description = 'View running ghost processes';
 LsCommand.global = true;
+LsCommand.aliases = ['status'];
```

Comment:
After seeing #375 I see how it could be _really_ confusing - I didn't realize the implications of aliases in individual command files until _after_ that PR was merged 😬